README.md

```diff
@@ -1 +1,40 @@
-# snips-react
\ No newline at end of file
+# snips-react
+
+This VS Code extension adds snippets for React in your editor for typescript.
+
+### TypeScript React Snippets (.tsx)
+
+| Snippet                      | Purpose                                     |
+|------------------------------|---------------------------------------------|
+| `react`                      | Snippet for default import (React & Radium) |
```

Comment:
Please add more snippets as discussed and accordingly change this snippet text. People might use the word `react` at other places as well.

Reply:
Fixed.